.docker/web/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM php:7.4-fpm
```

Comment:
Is this meant to be part of this changeset?

Reply:
I believe so - think it was just a newline thing